### PR TITLE
remove regular expressions

### LINKED ISSUE #652: Resolve profile string grammar
BLOCKER: The first bullet on page 35 has a normative statement about the AVT profile strings that must be accepted, but uses an unspecified grammar to actually specify the string. If this is normative, it needs to be formal. Either specify the string using a formal and cited grammar, or provide an exhaustive list.

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #652

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1785,17 +1785,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
           following rules when processing the media m= sections in a
           received offer:</t>
 
           <t>
             <list style="symbols">
-              <t>Any profile in the offer matching the following patterns
-              MUST be accepted: "RTP/[S]AVP[F]" and
-              "(UDP|TCP)/TLS/RTP/SAVP[F]".
-              </t>
+              <t>Any profile in the offer matching one of the following
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: For utmost clarity, perhaps do this as a list?

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -1785,17 +1785,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+              MUST be accepted:
+              "RTP/AVP", "RTP/AVPF", "RTP/SAVP", "RTP/SAVPF",
+              "UDP/TLS/RTP/SAVP", "UDP/TLS/RTP/SAVPF",
+              "TCP/TLS/RTP/SAVP", or "TCP/TLS/RTP/SAVPF".</t>
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: I note that this excludes:
UDP/TLS/RTP/AVP[F]

This is actually a normative change, though I guess a safe one.

Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: I'm going to add in UDP/TLS/RTP/AVP[F] - no idea if it is used but if it is , we should match it

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -1785,16 +1785,27 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
 
           <t>
             <list style="symbols">
-              <t>Any profile in the offer matching the following patterns
-              MUST be accepted: "RTP/[S]AVP[F]" and
-              "(UDP|TCP)/TLS/RTP/SAVP[F]".
+              <t>Any profile in the offer matching one of the following
+              MUST be accepted:
+              <list style="symbols">
+                <t> "RTP/AVP", </t>
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Suggest removing the punctuation - not needed for the list form.

**Pull request merged**